UTCP/src/ut_clnt.cpp

```diff
@@ -638,18 +638,13 @@ int CUT_WSClient::SetSecurityMode(SSLMode mode){
 
     m_sslMode = mode;
     switch(mode) {
+        case TLSv1:
+        case TLSv11:
+        case TLSv12:
         case TLS:
-            m_meth = TLSv1_client_method();
-            break;
-        case SSLv2:
-            m_meth = SSLv2_client_method();
-            break;
-        case SSLv3:
-            m_meth = SSLv3_client_method();
-            break;
-        case SSLv23:
         default:
-            m_meth = SSLv23_client_method();
+            //all version specific methods are deprecated since openssl 1.1.0

```

Comment:
Do we even need the enum then? Let's just remove this and hardcode the method.

Reply:
Yes, I also think it could be removed.